internal/keycloakb/accreditationsclient/accreditationsclient.go

```diff
@@ -26,6 +26,21 @@ type UpdateNotificationRepresentation struct {
 	UpdatedFields []string `json:"updatedFields,omitempty"`
 }
 
+// CreationCheckRepresentation struct
+type CreationCheckRepresentation struct {
```

Comment:
This second structure is not really useful.
The initial one was enough and supports both cases